psc-docs/Ctags.hs

```diff
@@ -1,5 +1,6 @@
 module Ctags (dumpCtags) where
 
+import Prelude
```

Comment:
This bug happens with `ghcid` by the way, so i'd suggest adding them these imports in a few more places just so both `intero` and `ghcid` will work. I agree that this is ugly and I can remove if you wish but the tradeoff of having them worth it in my opinion. Your call :)